static/app/components/onboarding/consoleModal.tsx

```diff
@@ -32,59 +33,57 @@ export const CONSOLE_PLATFORM_INSTRUCTIONS = {
-          "You can get started using Sentry on Nintendo Switch without any changes to your game, on devkits as well as retail devices. You can configure Nintendo's CRPORTAL to forward crashes to Sentry using [lp1Link:lp1 for retail devices] or [dd1Link:dd1 for devkits].",
-          {
-            lp1Link: (
-              <ExternalLink href="https://crash-report.wc.lp1.er.srv.nintendo.net/sentry/get_started" />
-            ),
-            dd1Link: (
-              <ExternalLink href="https://crash-report.wc.dd1.er.srv.nintendo.net/sentry/get_started" />
-            ),
-          }
+        {t(
+          'You can get started using Sentry on Nintendo Switch without any changes to your game, on DevKits as well as Retail devices.'
         )}
       </p>
-      <Heading as="h4">Nintendo Switch SDK</Heading>
       <p>
         {tct(
-          'To add more context to your crash dumps or capture non-fatal events, Sentry provides a dedicated SDK for Nintendo Switch. With this SDK, you can include details like [breadcrumbsLink:breadcrumbs] and [tagsLink:tags]. To request access, please use the [nintendoDeveloperAuthorizationFormLink:Nintendo Developer Authorization form].',
+          "Please complete the verification process by submitting an access request to the [nintendoDeveloperAuthorizationFormLink: Nintendo Developer Authorization form] and configure Nintendo's CRPORTAL to forward crashes to Sentry using [lp1Link:lp1 for retail devices] or [dd1Link:dd1 for devkits].",
           {
             nintendoDeveloperAuthorizationFormLink: (
               <ExternalLink href="https://developer.nintendo.com/group/development/getting-started/g1kr9vj6/middleware/sentry" />
             ),
-            breadcrumbsLink: (
-              <ExternalLink href="https://docs.sentry.io/product/issues/issue-details/breadcrumbs/" />
+            lp1Link: (
+              <ExternalLink href="https://crash-report.wc.lp1.er.srv.nintendo.net/sentry/get_started" />
             ),
-            tagsLink: (
-              <ExternalLink href="https://docs.sentry.io/product/issues/issue-details/#tags" />
+            dd1Link: (
+              <ExternalLink href="https://crash-report.wc.dd1.er.srv.nintendo.net/sentry/get_started" />
             ),
           }
         )}
       </p>
-      <p>{t('Sentry supports both the original Switch and Switch 2.')}</p>
-    </Fragment>
+      <p>{t("We'll receive your request and get back to you with the next steps.")}</p>
+      <Alert type="info" showIcon>
+        {t('Sentry supports both the original Switch and Switch 2.')}
+      </Alert>
+    </Flex>
   ),
   [ConsolePlatform.XBOX]: (
-    <Fragment>
+    <Flex direction="column" gap="md">
       <p>
-        {t('Sentry supports Xbox One and Series X|S, devkits as well as retail devices.')}
+        {t(
+          'You can get started using Sentry on Xbox without any changes to your game — including on Xbox One and Series X|S, across both DevKits and Retail devices.'
```

Comment:
This is incorrect. Only PlayStation and Nintendo Switch can be used 'without any changes to the game'. In other words, the SDK is optional, to capture non-fatal events + Add tags/crumbs/context to crash dumps.

For Xbox, you need the SDK anything to work:
```suggestion
          'Sentry supports Xbox One and Series X|S, across both DevKits and Retail devices through an SDK.'
```